test/SharpReverseProxy.Tests/HttpContextFakes/FakeHttpMessageHandler.cs

```diff
@@ -1,13 +1,21 @@
 using System.Net;
 using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace SharpReverseProxy.Tests.HttpContextFakes {
     public class FakeHttpMessageHandler : HttpMessageHandler {
         public HttpRequestMessage RequestMessage { get; private set; }
 
-        public HttpResponseMessage ResponseMessageToReturn { get; set; } = new HttpResponseMessage(HttpStatusCode.OK);
+        public HttpResponseMessage ResponseMessageToReturn { get; set; } = new HttpResponseMessage { 
```

Comment:
Hi @gbiellem , 
This is a general purpose class for testing any request/response, even the ones with other content types. 
What do you think about moving this logic to the test itself?

Reply:
Makes sense. , I'll move it